index.bs

```diff
@@ -11792,11 +11786,8 @@ in which case they are exposed on every object that [=implements=] the interface
         1.  If |attribute|'s type is a [=promise type=], then return
             [=!=] <a abstract-op>Call</a>({{%Promise.reject%}}, {{%Promise%}}, «|E|»).
         1.  Otherwise, end these steps and allow the exception to propagate.
-    1.  Let |F| be [=!=] <a abstract-op>CreateBuiltinFunction</a>(|steps|, « », |realm|).
-    1.  Let |name| be the string "<code>get </code>" prepended to |attribute|'s [=identifier=].
-    1.  Perform [=!=] <a abstract-op>SetFunctionName</a>(|F|, |name|).
-    1.  Perform [=!=] <a abstract-op>SetFunctionLength</a>(|F|, 0).
-    1.  Return |F|.
+    1.  Let |name| be |attribute|'s [=identifier=].
+    1.  Return [=!=] [$CreateBuiltinFunction$](|steps|, 0, |name|, « », |realm|, {{%Function.prototype%}}, "<code>get</code>").
```

Comment:
```suggestion
    1.  Return [=!=] [$CreateBuiltinFunction$](|steps|, 0, |name|, « », |realm|,
        |realm|.\[[Intrinsics]].\[[{{%Function.prototype%}}]], "<code>get</code>").
```

We need to use the %Function.prototype% from _realm_, since when this operation is called there could well be no current realm.